src/main/webapp/i18n/de/global.json

```diff
@@ -1,7 +1,7 @@
 {
     "global": {
         "title": "Artemis",
-        "browsehappy": "Du benutzt einen <strong>veralteten</strong> Browser. Bitte <a href=\"http://browsehappy.com/?locale=de/\">aktualisiere deinen Browser</a>, um die Benutzerfreundlichkeit zu erhöhen.",
+        "browsehappy": "Du benutzt einen <strong>veralteten</strong> Browser. Bitte <a href=\"http://browsehappy.com/?locale=de/\">aktualisiere deinen Browser</a>, um die Bedienbarkeit zu verbessern.",
```

Comment:
Same here regarding the term change. I feel like it's shifting the meaning, and that "Benutzerfreundlichkeit" would be the direct translation from English of "user-friendliness".

Reply:
The word exists and is, according to duden, a synonym for Benutzerfreundlichkeit. So I would leave it as it is... I think browser version affects everything that is covered by the word "Bedienbarkeit".